docs/burger-discounts/index.md

```diff
@@ -0,0 +1,545 @@
+# Burger Discounts
+
+International Burger Day falls on Tuesday of next week, so Madame Jellobutter
+decides to celebrate it by running the classic "buy 2 burgers get 1 free"
+promotion. She clearly lays out the rules of this promotion:
+
+> Given an order comprising multiple items, find the price of the second most
+> expensive burger. Subtract this number from the pre-tax total of the order.
```

Comment:
Wouldn't it be something like this? To make it more generic:

```suggestion
> Given an order comprising multiple items, for every two burgers ordered find the
> price of the less expensive one. Subtract this number from the pre-tax total of the
> order.
```

Reply:
This makes more sense, but I would like to implement it in the next chapter